test/e2e/metering_manual_install_test.go

```diff
@@ -107,6 +110,111 @@ func testManualMeteringInstall(
 	assert.NoError(t, err, "capturing logs and uninstalling metering should produce no error")
 }
 
+func createNFSProvisioner(ctx *deployframework.DeployerCtx) error {
+	//Waiting for Metering pods
+	ctx.TargetPodsCount = nonHDFSTargetPodCount
+
+	_, err := createTestingNamespace(ctx.Client, nfsTestingNamespace)
+	if err != nil {
+		return err
+	}
+
+	files := map[string]string{
+		"pod":          "server.yaml",
+		"service":      "service.yaml",
+		"storageClass": "storageclass.yaml",
+	}
+	server := &corev1.Pod{}
+	service := &corev1.Service{}
+	storageClass := &storagev1beta1.StorageClass{}
+	for name, file := range files {
+		absFile := filepath.Join(repoPath, testNFSManifestPath, file)
+		switch name {
+		case "pod":
+			if err := deploy.DecodeYAMLManifestToObject(absFile, server); err != nil {
+				return err
+			}
+			ctx.Client.CoreV1().Pods(nfsTestingNamespace).Create(context.TODO(), server, metav1.CreateOptions{})
+			if err != nil && !apierrors.IsAlreadyExists(err) {
+				return err
+			}
+		case "service":
+			if err := deploy.DecodeYAMLManifestToObject(absFile, service); err != nil {
+				return err
+			}
+			ctx.Client.CoreV1().Services(nfsTestingNamespace).Create(context.TODO(), service, metav1.CreateOptions{})
+			if err != nil && !apierrors.IsAlreadyExists(err) {
+				return err
+			}
+		case "storageClass":
+			if err := deploy.DecodeYAMLManifestToObject(absFile, storageClass); err != nil {
+				return err
+			}
+			ctx.Client.StorageV1beta1().StorageClasses().Create(context.TODO(), storageClass, metav1.CreateOptions{})
+			if err != nil && !apierrors.IsAlreadyExists(err) {
+				return err
+			}
+		}
+	}
+
+	svc, err := ctx.Client.CoreV1().Services(nfsTestingNamespace).Get(context.TODO(), "nfs-service", metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
+		return err
+	}
+	if err != nil || svc.Spec.ClusterIP == "" {
+		err = wait.Poll(3*time.Second, 5*time.Minute, func() (bool, error) {
+			svc, err = ctx.Client.CoreV1().Services(ctx.Namespace).Get(context.TODO(), "nfs-service", metav1.GetOptions{})
+			if apierrors.IsNotFound(err) {
+				return false, err
+			}
+			if err != nil {
+				return false, nil
+			}
+			return svc.Spec.ClusterIP != "" , nil
```

Comment:
As mentioned offline, you need to run `go fmt` on this as verify is complaining about `return svc.Spec.ClusterIP != "" ,` have a comma between the two return types.

Reply:
Thanks!